### PR TITLE
fix: fail early if oke options can't be fetched

### PR DESCRIPTION
**Pull request by carlossg**
and don't hide errors

#### Submitter checklist

- [x] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

**Comment by carlossg** [comment]
@hhuiliu I'm trying the oke create cluster and it's failing executing this oci command, which is also failing in the command line

```
$ oci ce node-pool-options get --node-pool-option-id all
RequestException: ('Connection broken: IncompleteRead(0 bytes read, 109 more expected)', IncompleteRead(0 bytes read, 109 more expected))
```

**Comment by carlossg** [comment]
Also raised in https://github.com/oracle/oci-cli/issues/119

**Comment by carlossg** [comment]
I got the command to work after adding the OKE policy but the error is very confusing

```
Your tenancy must also have the following required policy statement defined in the root compartment of your tenancy here by a user with administrative privileges:

allow service OKE to manage all-resources in tenancy
```

File: pkg/jx/cmd/create_cluster_oke.go [hunk 1]
```diff
@@ -143,7 +143,8 @@ func (o *CreateClusterOKEOptions) createClusterOKE() error {
 	//we assume user has prepared the oci config file under ~/.oci/
 	imagesArray, kubeVersionsArray, shapesArray, latestKubeVersion, err := oke.GetOptionValues()
 	if err != nil {
-		fmt.Println("error")
+		fmt.Println("Error getting OKE option values, have you created the OKE policy? https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengpolicyconfig.htm")
+		return err
```
Comment (pmuir) [inline pkg/jx/cmd/create_cluster_oke.go]: This is not the right way to log error messages. use `errors.Wrapf`

**Pull request merged**